include/LeMonADE/updater/moves/MoveAddScMonomer.h

```diff
@@ -45,9 +45,11 @@ along with LeMonADE.  If not, see <http://www.gnu.org/licenses/>.
  * 
  * @todo Code doubling with MoveAddBccMonomer! Specialized class only neccessary to enable the features to handle the moves types differently.
  **/
-class MoveAddScMonomer:public MoveBase
+class MoveAddScMonomer : public MoveAddMonomerBase<MoveAddScMonomer>
```

Comment:
We should think about a standard naming convention. As for MoveLocalBase and its derived classes MoveLocalBcc and MoveLocalSc the parent class is named MoveAddMonomerBase and this class should named MoveAddMonomerSc instead of MoveAddScMonomer.